### DISCUSSION TITLE
Mouse wheel scrolling

### DISCUSSION THREAD

**Discussion by ghost** [fancyapps/ui#216] [Q&A]
When the html window is large, a scrollbar appears inside the fancybox and I can't scroll with the mouse wheel in any way, how can I make it possible to scroll with the mouse wheel?

**Comment by fancyapps** [comment]
Hi,

Please, create a demo (using any code Playground) where this issue could be observed.

**Comment by ghost** [comment]
Here is a link to see what the mouse wheel does not scroll
https://fancyapps.com/docs/ui/fancybox/plugins/html
![2](https://user-images.githubusercontent.com/42895871/151177654-1a71abe7-bbec-4af6-8c5f-ec0acc77bb87.png)
![1](https://user-images.githubusercontent.com/42895871/151177660-7035028c-afc5-4a30-b680-f879a9b876f4.png)

**Reply by fancyapps** [reply]
Thanks for the report, I'll look into this.

**Reply by fancyapps** [reply]
This issue has been fixed in v4.0.21. Thanks again!

**Comment by ghost** [comment]
Thank you for your efficiency
